pkg/operator/ceph/csi/csi-rbdplugin-provisioner-dep.yaml

```diff
@@ -0,0 +1 @@
+../../../../cluster/examples/kubernetes/ceph/csi/template/rbd/csi-rbdplugin-provisioner-dep.yaml
```

Comment:
these links are needed for unit tests?

Reply:
no idea, I have just changed the file names as it was present already before this PR. let me test is it required or not